packages/cosmos_ui_components/pubspec.lock

```diff
@@ -69,7 +69,7 @@ packages:
     description:
       path: "packages/cosmos_utils"
```

Comment:
we should not remove all the `.fvm/fvm_config.json` files. those are needed to make sure we're all using the same version of flutter. if its outdated and differs from the one in `pull_request.yaml` then let's update those

Reply:
Sure